### DISCUSSION TITLE
Release 2021.3.4

### DISCUSSION THREAD

**Discussion by seanbudd** [nvaccess/nvda#13497] [General]
Highlights and download links can be found in the release blog post at: https://www.nvaccess.org/post/nvda-2021-3-4/

<hr /><em>This discussion was created from the release <a href='https://github.com/nvaccess/nvda/releases/tag/release-2021.3.4'>Release 2021.3.4</a>.</em>

**Comment by feerrenrut** [comment]
Also, for this release, please see https://github.com/nvaccess/nvda/security/advisories
